src/visitors/pages/Login.js

```diff
@@ -0,0 +1,255 @@
+/*
+ *           ---- Scan QR code ---                       ---- Load activities ---- Register visit
+ *          /                     \                    /
+ * Login --                        -- Get visitor ID --
+ *          \                     /                    \
+ *           --- Enter details ---                        ---- Failure screen
+ */
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import styled from 'styled-components';
+import { Row, Col, Grid } from 'react-flexbox-grid';
+import { BeatLoader as Bl } from 'react-spinners';
+import PrivacyStatement from '../components/PrivacyStatement';
+import QrScanner from '../components/QrScanner';
+import SignInForm from '../components/SignInForm';
+import SignInActivitiesGrid from '../components/SignInActivitiesGrid';
+import { Activities, Visitors, CbAdmin, CommunityBusiness } from '../../api';
+import { Paragraph } from '../../shared/components/text/base';
+import { PrimaryButton } from '../../shared/components/form/base';
+import NavHeader from '../../shared/components/NavHeader';
+import { redirectOnError } from '../../util';
+import { colors } from '../../shared/style_guide';
+
+
+const StyledSection = styled.section`
+  margin: ${props => props.margin}rem 0;
+  display: flex;
+  justify-content: center;
+`;
+
+const BeatLoader = styled(Bl)`
+  width: 60px;
+  margin: 0 auto;
+  padding-top: 45vh;
+`;
+
+const capitaliseFirstName = name => name.split(' ')[0].replace(/\b\w/g, l => l.toUpperCase());
+const greetingWithOrgName = orgName => `Welcome${orgName ? ` to ${orgName}` : ''}!`;
+
+export default class Login extends Component {
```

Comment:
Can this component be renamed to `SignIn`. `LogIn` is misleading to it's purpose.